tools/build-neatvnc.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Update and install dependencies
+sudo apt update
+sudo apt install -y meson ninja-build libpixman-1-dev  zlib1g-dev libdrm-dev pkgconf
+
+# Clone and build aml
+git clone https://github.com/any1/aml.git
+cd aml
+meson build
+ninja -C build
+sudo ninja -C build install
```

Comment:
Build statically-linked libraries when possible. In addition, eliminate the use of `sudo`.